components/Header.js

```diff
@@ -1,114 +1,105 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
 import Link from "next/link";
 import { withSession } from "../hooks/withSession";
```

Comment:
Please remove this unused import statement

Reply:
I have removed it.